index.js

```diff
@@ -179,8 +183,9 @@ class ShopifyToken {
             error.statusCode = status;
             return reject(error);
           }
-
-          resolve(body.access_token);
+          
+          const tokenData = shouldReturnAllData ? body : body.access_token;
```

Comment:
added this to support backwards compatibility

Reply:
I would prefer to return all data in all cases (no `shouldReturnAllData`) and cut a new major.